src/RE.cc

```diff
@@ -189,17 +189,24 @@ std::string Specific_RE_Matcher::LookupDef(const std::string& def) {
     return {};
 }
 
-bool Specific_RE_Matcher::MatchAll(const char* s) { return MatchAll((const u_char*)(s), strlen(s)); }
+bool Specific_RE_Matcher::MatchAll(const char* s) { return MatchAll(std::string_view{s}); }
 
 bool Specific_RE_Matcher::MatchAll(const String* s) {
     // s->Len() does not include '\0'.
     return MatchAll(s->Bytes(), s->Len());
```

Comment:
We could clean this up as well

```suggestion
    return MatchSet(s->ToStdStringView(), matches);
```